GAE/src/org/akvo/flow/domain/DataUtils.java

```diff
@@ -28,15 +31,17 @@
  */
 public class DataUtils {
 
+    private static final Logger log = Logger.getLogger(DataUtils.class);
+
     public static final ObjectMapper JSON_OBJECT_MAPPER = new ObjectMapper();
 
     public static String[] optionResponsesTextArray(String optionResponse) {
         String[] responseArray = null;
 
-        if (optionResponse == null || optionResponse.trim() == "") {
+        if (optionResponse == null || optionResponse.trim().equals("")) {
```

Comment:
Yes but in this case calling trim() may still produce NPE so the null check is required